storage/cloud-client/iam.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Copyright 2017 Google, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to get and set IAM policies on Google
+Cloud Storage buckets.
+
+For more information, see the documentation at
+https://cloud.google.com/storage/docs/access-control/using-iam-permissions.
+"""
+
+import argparse
+import base64
+import os
+
+from google.cloud import storage
+
+
+def view_bucket_iam_members(bucket_name):
+    storage_client = storage.Client()
+    bucket = storage_client.bucket(bucket_name)
+
+    policy = bucket.get_iam_policy()
+
+    for role in policy:
+        members = policy[role]
```

Comment:
Oh, ick.

If beginners like me are likely to read this code, maybe adding a comment about that here would prevent confusion. Up to you though, I don't have a lot of context yet.